### PR TITLE
Automatic commit of package [pdc-client] minor release [1.8.0-1].

### PR DESCRIPTION
**Pull request by xiangge**
Created by command:

/usr/bin/tito tag --keep-version

### CODE REVIEW

File: pdc-client.spec [hunk 1]
```diff
@@ -231,6 +231,30 @@ EOF
...
+- Update documentation (lholecek@redhat.com)
+- Add documentation link to README file (lholecek@redhat.com)
+- Add discription of page_size=-1 in help doc (chcao@redhat.com)
+- Fix printing errors and exit code for pdc_client (lholecek@redhat.com)
+- Add smoke-test for all executables (lholecek@redhat.com)
+- Fix running tests when building rpm (lholecek@redhat.com)
+- Override plugin paths with PDC_CLIENT_PLUGIN_PATH (lholecek@redhat.com)
+- Improve installing plugins (lholecek@redhat.com)
+- Revert removing comments from downstream (lholecek@redhat.com)
+- Automatic commit of package [pdc-client] minor release [1.7.0-2].
```
Comment (hluk) [inline pdc-client.spec]: Remove this line.

Comment (xiangge) [reply pdc-client.spec]: I saw this commit was committed after the 1.7.0-3 and also after the tag, do we need to keep this?

Comment (hluk) [reply pdc-client.spec]: Remove it. The commit is also in release branch, not sure how it got in master before.

Tito adds all new commits. It only tries to help you write better changelog. Some chagelog entries are only about refactoring or fixing issues that did not appear in previous versions (happened during development of new version) - these are not relevant to the last version.

Comment (xiangge) [reply pdc-client.spec]: Yes, I know that.

Just thinking to keep the log here for discussion if there is a need to keep, I also prefer to remove this.

**Review by hluk — APPROVED** [review]

**Review by simozhan — APPROVED** [review]

**Pull request merged**